include/vapor/ConstantGrid.h

```diff
@@ -50,30 +50,29 @@ class VDF_API ConstantGrid : public Grid {
     // They do nothing and return meaningless values.
     // Do not use!
     //
-    std::vector<size_t>        GetCoordDimensions(size_t) const override;
```

Comment:
Any idea why all of this code changed?

Reply:
clang-format did it

Reply:
OK. Any idea why? I.e. our entire code base was supposed to be consistently formatted with clang-format. Do you have any idea why clang-format decided to re-format this unchanged code now?

Reply:
There was one function signature change, which resulted a longer line. Clang-format tries to align all function names, so it shifted all function signatures, resulting the changes we see here.

Reply:
Got it. Thanks. Not sure I like that feature, but I can live with it :-)